### ISSUE TITLE
New design for Neuron.

### ISSUE THREAD

**Issue by yanguoyu** [Magickbase/neuron-public-issues#45]
- [ ] Dark Mode (1 week)
- [x] Import/Create Wallet (1.5 week)
- [x] Overview (1 week)
- [x] Send (1 week)
- [x] Receive (1 week)
- [x] History (1 week)
- [x] Nervos DAO (2 weeks)
- [x] Customized Assets (2 weeks)
- [x] Asset Accounts (2 weeks)
- [x] Setting (1 week)
- [x] Multisig Address (1 week)
- [x] Sign/Verify Message (0.5 week)
- [x] Offline Sign (0.5 week)

UI Design: https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=412%3A5634&t=ihXdpM6DFztFl5ag-0

**Comment by yanguoyu** [comment]
@Keith-CY Can you create a new branch for developing the new design? I will open a new issue related to this when developing an item in the list.

**Comment by Keith-CY** [comment]
> @Keith-CY Can you create a new branch for developing the new design? I will open a new issue related to this when developing an item in the list.

Sure, https://github.com/nervosnetwork/neuron/tree/new-ui-design

**Comment by yanguoyu** [comment]
https://github.com/Magickbase/visual-style-guide/issues/2

**Comment by Keith-CY** [comment]
Please add time estimation of each task

**Comment by yanguoyu** [comment]
> Please add time estimation of each task

I will add the estimated time when creating a new task. Or should I estimate all before starting it?

**Comment by Keith-CY** [comment]
> > Please add time estimation of each task
> 
> I will add the estimated time when creating a new task. Or should I estimate all before starting it?

We can estimate them at the very beginning as a reference for work arrangement but it's not required to be very precise and can be updated later.

Also, we could share some tasks among ourselves based on the time estimation.
